README.md

```diff
@@ -44,6 +44,12 @@ const tree = dirTree('./test/test_data', {extensions:/\.txt$/}, (item, PATH) =>
 
 The callback function takes the directory item (has path, name, size, and extension) and an instance of [node path](https://nodejs.org/api/path.html).
 
+## Options
+
+`exclude` : `RegExp` - A RegExp to test for exlusion of directories.
```

Comment:
Let's do the same for `exclude` for the sake of consistency. Thanks!

Reply:
You´re right. I slipped with the line, will correct it.